Demo/Configuration.swift

```diff
@@ -18,17 +18,17 @@ internal enum Configuration {
 
     static let appName = "Adyen Demo"
 
-    static let amount = Payment.Amount(value: 17408, currencyCode: "EUR")
+    static let amount = Payment.Amount(value: 17408, currencyCode: "JPY")
```

Comment:
Let's uncommit that :D

Reply:
its fine, its not a secret, right? when the PR is done, I will revert it.

Reply:
oh, it is WIP, sorry 😅